lib/apiservers/engine/backends/container.go

```diff
@@ -58,6 +58,19 @@ import (
 
 const (
 	bridgeIfaceName = "bridge"
+
+	// MemoryMinMiB - the minimum allowable container memory size
+	MemoryMinMiB = 64
+	// MemoryMaxMiB - the maximum allowable container memory size
+	MemoryMaxMiB = 16384
```

Comment:
That's a great idea, though I think we wanted to keep this initial stab pretty simple. Could we attack that question at a future date? On that subject, I wonder if we can make the memory max simple by just overcommitting on everything and letting ESX handle it. There's some interesting literature on the subject [here](https://labs.vmware.com/vmtj/memory-overcommitment-in-the-esx-server).